src/interfaces/vault.ts

```diff
@@ -411,6 +411,10 @@ export class VaultInterface<T extends ChainId> extends ServiceInterface<T> {
     }
   }
 
+  async findMetadataOverrides(address: Address): Promise<VaultMetadataOverrides> {
```

Comment:
Would prefer users fetch dynamic data (which includes Metadata) through `vaults.getDynamic()` rather than exposing a new method.

Reply:
Sounds reasonable, I'll look into this. I thought that function was only for the Vaults (in the VaultService) and not for the ones in the LabService. In there, we are fetching the vaults from this endpoint https://api.yearn.finance/v1/chains/1/vaults/all

Are we to refactor and fetch from `vault.getDynamic()`?

Reply:
Removed that function with https://github.com/yearn/yearn-sdk/pull/263/commits/69fa49237a012eb0028d8fb6e5f968853aef1670